csharp/ql/test/query-tests/API Abuse/DisposeNotCalledOnException/options

```diff
@@ -1,2 +1,2 @@
-semmle-extractor-options: /r:System.ComponentModel.Primitives.dll /r:${testdir}/../../../resources/assemblies/System.Data.dll /r:System.Data.Common.dll
```

Comment:
Can we now remove some of these DLLs?

Reply:
Yes, that is actually possible. I will narrow the list to only what is strictly needed.

Reply:
Ah - do you mean whether we can remove some of the DLLs from the repo?

Reply:
Yes, that's what I meant.